appeals/api/src/server/tests/appeals/expectation.js

```diff
@@ -38,14 +71,44 @@ export const baseExpectedLPAQuestionnaireResponse = (appeal) => ({
  */
 export const baseExpectedAppellantCaseResponse = (appeal) => {
 	return {
-		...formatAppellantCase(appeal),
+		...formatAppellantCase(appeal), // TODO: shouldn't be calling this from tests
```

Comment:
TODO not clear

Reply:
Added more detail as noted in my other reply

Reply:
it does! We really need some good refactoring with models, mocks and definitions...https://pins-ds.atlassian.net/browse/A2-456?focusedCommentId=72717

Reply:
I've opened a tech debt ticket to address this, and updated the TODO comments to reference it: https://pins-ds.atlassian.net/browse/A2-770